lxd/instance/drivers/driver_lxc.go

```diff
@@ -2690,6 +2690,12 @@ func (d *lxc) Stop(stateful bool) error {
 			op.Done(err)
 			return err
 		}
+
+		// Load the config.
+		err = d.c.LoadConfigFile(filepath.Join(d.LogPath(), "lxc.conf"))
+		if err != nil {
+			return fmt.Errorf("Failed to load raw.lxc")
```

Comment:
Please could you improve the error to include the error message and the path of the file being loaded?

Reply:
Also adding a comment as to why this is needed would be good for future reference, something along the lines that certain config keys are used at instance stop time (such as `lxc.signal`).